owl/EASE-STATE.owl

```diff
@@ -151,22 +183,13 @@ Several other Description subclasses may function as Configurations. For example
 
     <owl:Class rdf:about="http://www.ease-crc.org/ont/EASE-STATE.owl#Containment">
         <rdfs:subClassOf rdf:resource="http://www.ease-crc.org/ont/EASE-STATE.owl#FunctionalControl"/>
-        <rdfs:comment>Classifies States in which an object is placed inside another.</rdfs:comment>
-    </owl:Class>
-    
-
-
-    <!-- http://www.ease-crc.org/ont/EASE-STATE.owl#ContainmentConfiguration -->
-
-    <owl:Class rdf:about="http://www.ease-crc.org/ont/EASE-STATE.owl#ContainmentConfiguration">
-        <rdfs:subClassOf rdf:resource="http://www.ease-crc.org/ont/EASE-STATE.owl#FunctionalControlConfiguration"/>
         <rdfs:subClassOf>
             <owl:Restriction>
-                <owl:onProperty rdf:resource="http://www.ontologydesignpatterns.org/ont/dul/DUL.owl#definesRole"/>
+                <owl:onProperty rdf:resource="http://www.ease-crc.org/ont/EASE-STATE.owl#isGestalltOf"/>
```

Comment:
Ah right, so since StateTypes/Gestallts are concepts, they should not use "defines" when linking to Roles (the way Situations or Descriptions might). Ok, makes sense.